backend/tests/Integration/Request/RequestDashboardSnapshotTest.php

```
@@ -0,0 +1,121 @@
+    }
+
+    public function testExistingTransactionRemainsCallerOwned(): void
+    {
+        $db = $this->connection();
+        $db->open();
+        $managerId = $this->managerId($db);
+        $transaction = $db->beginTransaction(Transaction::READ_COMMITTED);
+
+        (new RequestDashboardQuery($db))->findFor($managerId);
+
+        self::assertTrue($transaction->isActive);
```

PHPStan (2.2.7)

[error] Call to an undefined static method Tests\Integration\Request\RequestDashboardSnapshotTest::assertTrue().


(staticMethod.notFound)